lib/api/controllers/serverController.js

```diff
@@ -100,6 +101,24 @@ class ServerController {
   /**
    * @returns {Promise<Object>}
    */
+  healthCheck() {
+    return this.kuzzle.services.list.memoryStorage.getInfos()
+      .then(() => this.kuzzle.services.list.internalCache.getInfos())
+      .then(() => this.kuzzle.services.list.storageEngine.getInfos())
+      .catch(err => {
+        throw new ExternalServiceError(err);
+      })
+      .then(response => {
+        if (['yellow', 'green'].indexOf(response.status) < 0) {
+          throw new ExternalServiceError('es is down: ' + JSON.stringify(response));
```

Comment:
Nitpick: Can we return "Elasticsearch is down" instead of "es"? If a user is not comfortable with Kuzzle stack, he probably doesn't know what "es" means.